### PR TITLE
MatterMost fixed the eldap new SECTIONS branch

### PR DESCRIPTION
**Pull request by ubc-tuehoang**
updated code to fix the new ELDAP structure with SECTIONS:

- Updated the filter for eldap search with the new SECTIONS 
- Updated the components to respective variables when synch call: 
        - [courseCODE] [courseNumber] [courseSectionNum] [year] [students (default) / instructor]
        - example: 211 101 03 2024 instructors
- Updated the uniqueMember parameter based on the revised search query—retaining only uid=username.

### CODE REVIEW

File: mattermostsync/__init__.py [hunk 1]
```diff
@@ -83,18 +105,26 @@ def get_member_from_ldap(self, base, course, campus='UBC', attributes=LDAP_ATTRI
                 member_emails.append(m['email'])
 
         self.logger.info('Get {} student in course {} {}'.format(len(members), course, campus))
-        self.logger.debug('Students:' + str(members))
+        self.logger.debug('Studentsssss:' + str(members))
```
Comment (xcompass) [inline mattermostsync/__init__.py]: is this a typo?

File: run-SECTIONS.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+python ./sync.py \
```
Comment (xcompass) [inline run-SECTIONS.sh:19]: Could you add some comment about this script? What is is used for? Is it just for testing a single section sync?

**Review by xcompass — CHANGES_REQUESTED** [review]
Looks good. just need minor changes. Thanks.

File: mattermostsync/__init__.py (hunk 1 above)
Comment (ghost) [reply mattermostsync/__init__.py]: it was a flag to check during debug - will remove it.

File: run-SECTIONS.sh (hunk 2 above)
Comment (ghost) [reply run-SECTIONS.sh:19]: It was extracted from the README for a quick test. I'll also add comments to the script.

**Comment by ubc-tuehoang** [comment]
Changes are complete, so I will close this pull request and create a new one with the latest updates.

**Pull request closed without merging**